lib/clientv2/ippool_e2e_test.go

```diff
@@ -399,20 +426,7 @@ var _ = testutils.E2eDatastoreDescribe("IPPool tests", testutils.DatastoreAll, f
 				},
 				{
 					Type:   watch.Added,
-					Object: outRes3,
-				},
-			})
-
-			By("Cleaning the datastore and expecting deletion events for each configured resource (tests prefix deletes results in individual events for each key)")
-			be.Clean()
-			testWatcher4.ExpectEvents(apiv2.KindIPPool, []watch.Event{
-				{
-					Type:     watch.Deleted,
-					Previous: outRes1,
-				},
-				{
-					Type:     watch.Deleted,
-					Previous: outRes3,
+					Object: outRes4,
```

Comment:
I've removed the last test - it'll be a bit of a pain to update, and we already test the delete everything case in all of the other resource types - so we don't gain anything by testing for this resource too.